Controllers/AdminController.js

```diff
@@ -0,0 +1,177 @@
+const AsyncHandler = require("express-async-handler");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const Admin = require("../Models/AdminModel");
```

Comment:
have a single line space between npm packages and your packages. Again just better readability